src/python/runFacetsBenchmark.py

```diff
@@ -54,8 +54,17 @@ def run_benchmark(lucene_dir, index_dir, data_dir, nightly_log_dir, doc_limit, n
     print('Indexing documents...\n')
     subprocess.check_call(index_cmd, shell=True)
 
+    hppc_path = None
+    for root_path, dirs, files in os.walk(os.path.expanduser('~/.gradle/caches/modules-2/files-2.1/com.carrotsearch/hppc')):
+        for file in files:
+            if file == 'hppc-0.9.0.jar':
```

Comment:
Do we want to version lock this? Or maybe we should do something like `if 'hppc' in file` (or regex to be more accurate)?

Reply:
This is the same (hacky!) logic we use in `benchUtil.py` -- for now let's keep it, though it would be better to factor it out (later).